docs/sources/static/configuration/integrations/azure-exporter-config.md

```diff
@@ -11,9 +11,16 @@ description: Learn about azure_exporter_config
 ## Overview
 The `azure_exporter_config` block configures the `azure_exporter` integration, an embedded version of
 [`azure-metrics-exporter`](https://github.com/webdevops/azure-metrics-exporter), used to
-collect metrics from [Azure Monitor](https://azure.microsoft.com/en-us/products/monitor). The
-exporter uses [Azure Resource Graph](https://azure.microsoft.com/en-us/get-started/azure-portal/resource-graph/#overview)
-queries to identify resources for gathering metrics.
+collect metrics from [Azure Monitor](https://azure.microsoft.com/en-us/products/monitor). 
+
+The exporter offers two options for gathering metrics,
+1. (Default) Use an [Azure Resource Graph](https://azure.microsoft.com/en-us/get-started/azure-portal/resource-graph/#overview) query to identify resources for gathering metrics
+   1. This will make 1 API call per resource identified
+   1. Subscriptions with a reasonable amount of resources are liable to hit the [12000 requests per hour rate limit](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/request-limits-and-throttling#subscription-and-tenant-limits) azure enforces 
+1. Set the regions to gather metrics from and get metrics for all resources across those regions
+   1. This will make an API call per subscription reducing the number of API calls dramatically 
+   1. This approach does not work with all resource types and Azure's does not document which resource types do/do not work
+   1. A resource type which is not support will produce errors which look like `Resource type: microsoft.containerservice/managedclusters not enabled for Cross Resource metrics`
```

Comment:
Love the detail. What is the recourse if a user hits this error?

Reply:
That's a great question, I think the only option is to fall back to the resource graph API based approach which I can add in this list to be clear.

Reply:
Awesome I think that would help!